<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>91</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>40</y>
      <width>85</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>9</y>
      <width>188</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Which page are you on now?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="offsetSpinBox">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>5</y>
      <width>60</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="minimum">
     <number>-16777215</number>
    </property>
    <property name="maximum">
     <number>16777215</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>85</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
